user-guide/Basic_Functionality/Protocols_and_templates/Alarm_templates/tutorials/Getting_started_with_alarm_templates.md

```diff
@@ -0,0 +1,220 @@
+
+1. In the *Versions* column, right-click the deployed version of the protocol and select *Set as production*.
+
+   Creating the alarm template for the production version will have the advantage that there will be no need to create a new alarm template for every new version of the protocol. Even when a new version is set as production, the same alarm template will continue to apply.
+
+1. Make sure the *Production* version is selected, right-click the *Elements* column on the right, and select *New element*.
+
+1. Create a new element with a name of your choice.
+
+   The production version of the *Generic Data Simulator* protocol should automatically be selected in the element editor.
+
+   > [!TIP]
+   > See also: [Adding elements](xref:Adding_elements)
+
+## Step 2: Configure alarm monitoring using normal thresholds
+
+1. Look up the newly created element in the Cube Surveyor.
+
+1. Right-click the element and select *Protocols & Templates* > *Assign alarm template* > *\<New alarm template\>*.
+
+1. In the dialog, keep the *Alarm template* option selected, specify the name *Default*, and click *OK*.
+
+1. Select the *CPU* parameter, and click *OK* at the bottom of the window.
+
+   In the element, you should now see the alarm severity color displayed next to the *CPU* parameter on the *General* page.
+
+1. Set *Simulation Mode* to a different value and check what happens.
+
+   The different modes simulate situations that can trigger alarms, so the alarm severity color next to *CPU* will change accordingly, and an alarm may be generated in the Alarm Console.
+
+![Create an alarm template](~/user-guide/images/KataAlarmTemplatesNewAlarmTemplate.gif)
+
+## Step 3: Configure hysteresis
+
+When *Simulation Mode* is set to *Spikes*, alarms will exist for short periods of time, which might make it difficult for operators to notice the alarms. The alarms might also not be relevant, as it could be that operators are only interested if the threshold is violated for longer periods of time. To deal with this, you can configure the hysteresis feature in the alarm template.
+
+1. Right-click the element in the Surveyor, and select *Protocols & Templates* > *View alarm template 'Default'*.
+
+   This will open the alarm template again.
+
+1. In the *HYST ON* column for the *CPU* parameter, specify *10*.
+
+   An alarm will now only be generated if the CPU value remains high for 10 seconds.
+
+1. In the *HYST OFF* column for the *CPU* parameter, specify *20*.
+
+   The alarm will now remain present for 20 seconds longer after the CPU value drops below the alarm threshold, ensuring that operators have more time to notice it.
+
+1. Click *OK* to save your changes.
+
+> [!NOTE]
+> Be mindful of the polling interval of the parameters when defining hysteresis. If your parameter is only updated every 5 seconds, you need to configure at least 10 seconds (two updates) for the hysteresis to make a difference.
```

Comment:
Is there any way users could easily find out what the polling interval of a parameter is? If a user doesn't know how to find this, they won't be able to follow the advice in the note.